api/modules/config.py

```diff
@@ -20,6 +20,51 @@ def define_string_list(name: str, args: bool=False, usage: str="") -> None:
       usage: When arg parsing fails, what to print for this setting's description.
     """
 
+def define_string(name: str, args: bool=False, usage: str="") -> None:
+    """
+    Defines a config setting of type `str`.
+
+    Allows the user invoking Tilt to configure a key named ``name`` to be in the
+    dict returned by :meth:`parse`.
+
+    To set a flag of this type named, e.g., `foo`, at runtime to "bar", run ``tilt up -- --foo bar``.
+
+    See the `Tiltfile config documentation <tiltfile_config.html>`_ for examples
+    and more information.
+
+    Args:
+      name: The name of the config setting
+      args: If False, the config setting is specified by its name. (e.g., if it's named "foo",
+            ``tilt up -- --foo bar`` this setting would be ``"bar"``.)
+
+            If True, the config setting is specified by unnamed positional args. (e.g.,
+            in ``tilt up -- 1``, this setting would be ``"1"``.)
+      usage: When arg parsing fails, what to print for this setting's description.
+    """
+
+def define_bool(name: str, args: bool=False, usage: str="") -> None:
+    """
+    Defines a config setting of type `bool`.
+
+    Allows the user invoking Tilt to configure a key named ``name`` to be in the
+    dict returned by :meth:`parse`.
+
+    To set a flag of this type named, e.g., `foo`, to True, run ``tilt up -- --foo``.
```

Comment:
see above, reorder this clause so "e.g." comes first

Reply:
also maybe worth giving an example of how to set to false cuz that's a little less intuitive